echo-webhooks/src/main/groovy/com/netflix/spinnaker/echo/scm/bitbucket/server/BitbucketServerEventHandler.java

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright 2022 Armory
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.echo.scm.bitbucket.server;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.netflix.spinnaker.echo.api.events.Event;
+import com.netflix.spinnaker.echo.jackson.EchoObjectMapper;
+import com.netflix.spinnaker.echo.scm.BitbucketWebhookEventHandler;
+import java.util.List;
+import java.util.Map;
+import org.springframework.stereotype.Component;
+
+@Component
+public class BitbucketServerEventHandler {
+
+  public static final List<String> bitbucketServerEventTypes =
+      List.of(
+          "pr:opened",
+          "repo:refs_changed",
+          "pr:from_ref_updated",
+          "pr:merged",
+          "pr:declined",
+          "pr:deleted");
+
+  private String repoProject;
+  private String slug;
+  private String hash;
+  private String branch;
+
+  private final ObjectMapper objectMapper = EchoObjectMapper.getInstance();
+
+  public void handleBitbucketServerEvent(Event event, Map postedEvent) {
```

Comment:
I think `postedEvent` parameter  is not used

Reply:
Good catch. Fixed.